modules/javafx.graphics/src/main/java/javafx/css/CssMetaData.java

```diff
@@ -327,5 +328,26 @@ public int hashCode() {
             .append("}").toString();
     }
 
-
+    /**
+     * Utility method which combines CssMetaData items in one unmodifiable list with the size equal to the number
+     * of items it holds (i.e. with no unnecessary overhead).
+     *
+     * @param list the css metadata items, usually from the parent, not nullable
+     * @param items the additional items
+     * @return the unmodifiable list containing all of the items
+     *
+     * @since 22
+     */
+    public static List<CssMetaData<? extends Styleable, ?>> initStyleables(
```

Comment:
I think this enhancement shouldn't be a part of this PR.

Reply:
created  [JDK-8320796](https://bugs.openjdk.org/browse/JDK-8320796)